### ISSUE TITLE
puppeteer no option to save passwords

### ISSUE THREAD

**Issue by leanderhamzoui** [puppeteer/puppeteer#9339]
Hello,

When submitting fields and chrome suggesting to save the login:pass to browser profile, there is no option in puppeteer to click "save" to store the login:pass to browser profile.

I also did not find any workaround on the web.

**Comment by jrandolf** [comment]
The suggestion to save the password is independent of the web page. Puppeteer has no access to this component, however we can consider implementing the CDP methods required for this.

**Comment by AndriusCTR** [comment]
> The suggestion to save the password is independent of the web page. Puppeteer has no access to this component, however we can consider implementing the CDP methods required for this.

Yes, please do this for CDP! It very popular component and we need to be able to control it (fill & save in the pop-up)
